openshift/default-catalog-consistency/pkg/extract/extract.go

```diff
@@ -69,6 +69,11 @@ func UnpackImage(ctx context.Context, imageRef, name string) (res *ExtractedImag
 			OSChoice: "linux",
 		}
 
+		if authPath := os.Getenv("REGISTRY_AUTH_FILE"); authPath != "" {
+			fmt.Println("Using registry auth file:", authPath)
+			sysCtx.AuthFilePath = authPath
+		}
```

Comment:
IMO, we shouldn't be handling envvars inside a library function because other potential callers of this function may want to configure auth file paths differently. It would be better if we handled this is the test entrypoint.

I know this is somewhat hypothetical right now, since there are no other callers. But we've discussed consolidating this with our other `containers/image` implementations, and I wouldn't want us to cargo-cult this into an actual shared library that we may create in the future.